articles/quickstart/spa/react/01-login.md

```diff
@@ -79,7 +79,7 @@ Add the `LoginButton` component to your application. When you click it, verify t
 Once that's complete, verify that Auth0 redirects you to your application using the value of the `redirectUri` that you used to configure the `Auth0Provider`.
 :::
 
-![Auth0 Universal Login](https://cdn.auth0.com/blog/universal-login/lightweight-login.png)
+![Auth0 Universal Login](/media/articles/angularjs/universal-login.png)
```

Comment:
To avoid confusion, as this same image looks to be used across multiple technologies and quickstarts, consider moving the image out to a common directory instead of being tied to a specific tech. It'll additionally save some grokking later if the image ever has to change and will be more obvious it's a shared image.